modules/capability/module.go

```diff
@@ -26,7 +26,6 @@ import (
 )
 
 var (
-	_ module.BeginBlockAppModule = (*AppModule)(nil)
```

Comment:
@tac0turtle I see there's now a [HasABCIEndblock](https://github.com/cosmos/cosmos-sdk/blob/v0.50.0-alpha.1/types/module/module.go#L221) interface, is there a similar interface for `BeginBlock` now or is it inferred somewhere else?

Reply:
I guess this is for legacy? and now core/appmodule is the std with [HasBeginBlocker](https://github.com/cosmos/cosmos-sdk/blob/7b1cd3c75afa92be01c10ab556fba04db3415c90/core/appmodule/module.go#L57) ?